isthmus/src/main/java/io/substrait/isthmus/SubstraitRelVisitor.java

```diff
@@ -32,13 +34,15 @@ public class SubstraitRelVisitor extends RelNodeVisitor<Rel, RuntimeException> {
   private final RexVisitor<Expression> converter;
   private final AggregateFunctionConverter aggregateFunctionConverter;
 
+  Map<RexFieldAccess, Integer> fieldAccessDepthMap;
```

Comment:
Why not private? Usually it is good to avoid package protected fields.

Reply:
change to `private`.